### ISSUE TITLE
Add option to set `aspectRatio` and `photoSize` on init

### ISSUE THREAD

**Issue by Zazo032** [Apparence-io/camera_awesome#174]
## Proposal

At least for now, there's no clear doc or way to set a specific initial aspect ratio and photo size

**Comment by Zazo032** [comment]
Probably not the best solution, but works until there's a way to do it on init:
```dart
CameraAwesome.custom(
  ...
  builder: (CameraState state) {
    ...
    final sensorConfig = snapshot.requireData;
    if (sensorConfig.aspectRatio != CameraAspectRatios.ratio_16_9) {
      sensorConfig.setAspectRatio(CameraAspectRatios.ratio_16_9);
      CamerawesomePlugin.setPhotoSize(1080, 1920);
    }
    ...
    return ...;
  }
),
```

**Comment by Zazo032** [comment]
@apalala-dev @g-apparence calling `await CamerawesomePlugin.setPhotoSize(1080, 1920)` seems to do nothing, is there any way to change the final photo size with the new implementation? Final images are 4000x2250, which is too big for our use case. Tried a couple different approaches, but I couldn't find a working one

**Comment by g-apparence** [comment]
Hi,
we made this on purpose. 

The new version doesn't include selecting the photo size because of 
- CameraX makes it impossible to choose ratio and size together
- Apps mainly use the best value the camera offers 
- We cannot garanty the size will be what you selected as cameraX / hardware will choose to use another one. 
- this makes the configuration a bit more complex when this is not really important 90% of the time

For now our approach is 
- take the best image 
- aspect ratio can be choose
- let the user resize / crop using the image package 

Would this fit your need?

**Comment by Zazo032** [comment]
@g-apparence Hi, makes sense the approach you followed. We implemented an extra step in our flow to compress images to our desired size. To set the aspect ratio, we're currently doing this:
```dart
CameraAwesomeBuilder.custom(
  builder: (state) {
    if (sensorConfig.aspectRatio != CameraAspectRatios.ratio_16_9) {
      sensorConfig.setAspectRatio(CameraAspectRatios.ratio_16_9);
      return ...
    }
  }
),
```
Might not be the best approach, but I think it's the only way to do it automatically right now.

**Comment by Zazo032** [comment]
Fixed by #180
